packaging/docker/sidecar/sidecar.py

```diff
@@ -0,0 +1,633 @@
+#! /usr/bin/env python3
```

Comment:
This is the only line I changed when copying this over from fdb-kubernetes-operator.